brushsettings.json

```diff
@@ -232,6 +232,15 @@
             "minimum": 0.0,
             "tooltip": "Hard brush-circle borders (setting to zero will draw nothing). To reach the maximum hardness, you need to disable Pixel feather."
         },
+        {
+            "constant": false,
+            "default": 0.5,
```

Comment:
This should be set to 0.0 to not break existing brushes.

Reply:
Ah.. yes, fixed.